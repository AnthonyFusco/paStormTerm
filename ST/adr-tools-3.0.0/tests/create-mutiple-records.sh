adr new First Decision
adr new Second Decision
adr new Third Decision
ls doc/adr
head -7 doc/adr/0002-second-decision.md
head -7 doc/adr/0003-third-decision.md
