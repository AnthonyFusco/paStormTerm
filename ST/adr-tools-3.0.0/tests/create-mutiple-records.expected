adr new First Decision
doc/adr/0001-first-decision.md
adr new Second Decision
doc/adr/0002-second-decision.md
adr new Third Decision
doc/adr/0003-third-decision.md
ls doc/adr
0001-first-decision.md
0002-second-decision.md
0003-third-decision.md
head -7 doc/adr/0002-second-decision.md
# 2. Second Decision

Date: 1992-01-12

## Status

Accepted
head -7 doc/adr/0003-third-decision.md
# 3. Third Decision

Date: 1992-01-12

## Status

Accepted
